packages/sync/src/modules/Comments.php

```diff
@@ -178,7 +180,7 @@ function filter_meta( $args ) {
 
 	public function expand_comment_ids( $args ) {
 		list( $comment_ids, $previous_interval_end ) = $args;
-		$comments    = get_comments(
+		$comments                                    = get_comments(
```

Comment:
oops

Reply:
This seems ok - see the statement above.

Reply:
true!